<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connections</class>
 <widget class="QDialog" name="connections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conex√µes</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="connectionslist"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add">
       <property name="text">
        <string>Incluir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connect">
       <property name="text">
        <string>Connectar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>connections</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
